subprojects/docs/src/docs/release/notes.md

```diff
@@ -61,6 +61,10 @@ You may now force Gradle to use rich or plain [build output](userguide/console.h
 
 The JaCoCo plugin has been upgraded to use [JaCoCo version 0.7.9](http://www.jacoco.org/jacoco/trunk/doc/changes.html) by default.
 
+### Adjustments in the `eclipse` plugin
+
+The `eclipse` plugin now defines separate output directories for each source folders. Also, each source folders and dependencies define an additional `gradle_source_sets` classpath attribute. The attribute contains a comma-separated list of source sets which contains the element. Future [Buildship](http://eclipse.org/buildship) versions will use this information to separate source sets when launching Java applications within Eclipse.
```

Comment:
The attribute specifies to which sourceSet the entry belonged.

Reply:
Fixed